packages/create-cloudflare/e2e/tests/frameworks/test-config.ts

```diff
@@ -625,6 +625,21 @@ function getFrameworkTestConfig(pm: string): NamedFrameworkTestConfig[] {
 			},
 			nodeCompat: false,
 		},
+		{
+			name: "waku",
+			testCommitMessage: true,
+			timeout: LONG_TIMEOUT,
+			unsupportedOSs: ["win32"],
+			verifyDeploy: {
+				route: "/",
+				expectedText: "Waku",
+			},
+			verifyPreview: {
+				route: "/",
+				expectedText: "Waku",
+			},
+			nodeCompat: false,
```

Comment:
Waku requires "nodejs_als" compatibility flag for dynamic server requests. 

https://github.com/wakujs/waku/blob/main/examples/07_cloudflare/wrangler.jsonc#L6-L8

https://github.com/cloudflare/workers-sdk/pull/8314/files#diff-d99ee22e8b983f8b81c51dc0c8a46d188f42df2c7273e2bc623f4efc4e108428R8

https://github.com/wakujs/waku/blob/main/docs/guides/cloudflare.mdx#notes-on-cloudflares-workerd-runtime